services/docs-snapcraft-io.yaml

```diff
@@ -39,6 +39,12 @@ spec:
                 secretKeyRef:
                   name: sentry-dsn
                   key: docs-snapcraft-io
+          env:
+            - name: SEARCH_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: google-api
+                  key: snapcraft-io-search-key
```

Comment:
Oh. This is merged. I was suggesting we didn't name this `snapcraft` at all. This key is for querying the Custom Search API for all sites - ubuntu.com and subdomains, snapcraft, canonical etc. And so I would expect that we'll use this key as-is in multiple sites. I think it should just be named `google-custom-search-key`.

Reply:
We haven't deployed it yet so I guess we can make a quick change in another PR. If the API key will be the same for all let's do it then

Reply:
https://github.com/canonical-webteam/deployment-configs/pull/207/files